# disable waf's python extension mode because it explicitly links libpython
# https://code.google.com/p/waf/issues/detail?id=1531
# from homebrew recipe 
# https://github.com/Homebrew/homebrew/blob/master/Library/Formula/py2cairo.rb

diff --git a/src/wscript b/src/wscript
index 69c6322..955ef49 100644
--- src/wscript
+++ src/wscript
@@ -19,7 +19,7 @@ def build(ctx):
 
   # C extension module
   ctx(
-    features     = 'c cshlib pyext',
+    features     = 'c cshlib',
     source       = ['cairomodule.c',
                     'context.c',
                     'font.c',
